src/core/devportal.tf

```diff
@@ -135,6 +135,9 @@ module "appservice_devportal_be" {
     JIRA_EMAIL_ID_FIELD        = "customfield_10084"
     JIRA_ORGANIZATION_ID_FIELD = "customfield_10088"
     JIRA_TOKEN                 = data.azurerm_key_vault_secret.devportal_jira_token.value
+
+    # Feature Flags
```

Comment:
Can we write a more descriptive comment? Like what kind of values we accept, what the behavior we expect, etc.
Pretend someone has to configure this application while you are on holiday